docs/contributing/README.md

```diff
@@ -93,25 +93,27 @@ For additional features and advanced configurations, refer to the official [MkDo
 
 ## Testing
 
-```bash
-pip install -r requirements/dev.txt
+??? note "Commands"
```

Comment:
Just my two cents, but I was looking at [the new page](https://docs.vllm.ai/en/latest/contributing/#clone-the-vllm-repository) and completely didn't notice this collapsed section and thought it was removed. 
<img width="730" alt="image" src="https://github.com/user-attachments/assets/cf52b2cd-fc2b-45fd-9731-66eac13a6dde" />